dev/Generated/TreeViewItem.properties.cpp

```diff
@@ -82,7 +82,7 @@ void TreeViewItemProperties::EnsureProperties()
                 winrt::name_of<double>(),
                 winrt::name_of<winrt::TreeViewItem>(),
                 false /* isAttached */,
-                ValueHelper<double>::BoxValueIfNecessary(12.0),
+                ValueHelper<double>::BoxValueIfNecessary(8.0),
```

Comment:
Why did this change?  Was this updated by a previous change but then not checked in?

Reply:
I think so, I'm getting this change when I build a clean version of Main.